src/aiida_quantumespresso/workflows/pdos.py

```diff
@@ -383,6 +397,9 @@ def setup(self):
         """Initialize context variables that are used during the logical flow of the workchain."""
         self.ctx.serial_clean = 'serial_clean' in self.inputs and self.inputs.serial_clean.value
         self.ctx.dry_run = 'dry_run' in self.inputs and self.inputs.dry_run.value
+        self.ctx.energy_range_vs_fermi = (
```

Comment:
Why do we have to add this to the context? Is it updated somehow?

Reply:
No, I think I just did at some point because it is used in the method to generate the inputs for the PDOS and in the one for the DOS inputs. But as it's not updated, I can also simply access the `self.inputs` directly. Will update this aspect and let you know so we can merge it

Reply:
Hi @mbercx, I just pushed the changes, should be fine now